src/pybind/mgr/dashboard/frontend/src/app/shared/directives/dimless-binary.directive.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Directive,
+  ElementRef,
+  EventEmitter,
+  HostListener,
+  Input,
+  OnInit,
+  Output
+} from '@angular/core';
+import { NgControl } from '@angular/forms';
+
+import * as _ from 'lodash';
+
+import { DimlessBinaryPipe } from '../pipes/dimless-binary.pipe';
+import { FormatterService } from '../services/formatter.service';
+
+@Directive({
+  selector: '[cdDimlessBinary]'
+})
+export class DimlessBinaryDirective implements OnInit {
+
+  @Output() ngModelChange: EventEmitter<any> = new EventEmitter();
+  @Input() minBytes: number;
```

Comment:
Can you please explain the input variables and how to use them, e.g. explain what values should/can be used.

Reply:
I've added documentation for component inputs.